plugins/c9.ide.terminal/terminal.js

```diff
@@ -64,6 +64,14 @@ define(function(require, exports, module) {
             "dark"  : ["#153649", "#FFFFFF", "#515D77", true],
             "dark-gray"  : ["#153649", "#FFFFFF", "#515D77", true]
         };
+        (function() {
+            var themeName;
+            if (options.defaults) {
+                for (themeName in options.defaults) {
+                    defaults[themeName] = options.defaults[themeName];
+                }
+            }
+        })();
```

Comment:
Please run any logic inside the "load" handler at https://github.com/c9/core/pull/137/files#diff-04e2cdcd86ed9a9c85ca77e05c506dfdR79. And there is no need to wrap this in a function.

Also, per [the coding conventions](https://github.com/ajaxorg/cloud9infra/blob/master/CODING_STANDARDS.md#variable-declarations), please don't use the Crockford/C style and avoid putting declarations at the top of a function. In this case, `themeName` should be declared in the `for` loop.